fluent-bundle/test/arguments_test.js

```diff
@@ -3,9 +3,9 @@
 import assert from 'assert';
 import ftl from "@fluent/dedent";
 
-import {FluentBundle} from '../esm/bundle';
-import {FluentResource} from '../esm/resource';
-import {FluentType, FluentNumber, FluentDateTime} from '../esm/types';
+import {FluentBundle} from '../esm/bundle.js';
```

Comment:
Question: If someone regresses this by changing it back to `'../esm/bundle'` will our CI catch it?

Reply:
Yes, the Mocha tests will fail then. The exact error they fail with is a bit misleading though:

```
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/eemeli/code/fluent-js/fluent-bundle/test/arguments_test.js
require() of ES modules is not supported.
require() of /Users/eemeli/code/fluent-js/fluent-bundle/test/arguments_test.js from /Users/eemeli/code/fluent-js/node_modules/mocha/lib/esm-utils.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename arguments_test.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /Users/eemeli/code/fluent-js/fluent-bundle/test/package.json.
```

There's a `try { ... } catch` block around the `import()` [here](https://github.com/mochajs/mocha/blob/master/lib/esm-utils.js#L47) which falls back to `require()`, and logs _its_ error if both fail. I may end up filing an upstream bug about this.